perfkitbenchmarker/providers/__init__.py

```diff
@@ -12,17 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+import os
+
 from perfkitbenchmarker import import_util
 
-from perfkitbenchmarker.providers.alicloud import ali_virtual_machine  # NOQA
-from perfkitbenchmarker.providers.aws import aws_virtual_machine  # NOQA
-from perfkitbenchmarker.providers.azure import azure_virtual_machine  # NOQA
-from perfkitbenchmarker.providers.cloudstack import cloudstack_virtual_machine  # NOQA
-from perfkitbenchmarker.providers.digitalocean import digitalocean_virtual_machine  # NOQA
-from perfkitbenchmarker.providers.gcp import gce_virtual_machine  # NOQA
-from perfkitbenchmarker.providers.kubernetes import kubernetes_virtual_machine  # NOQA
-from perfkitbenchmarker.providers.openstack import os_virtual_machine  # NOQA
-from perfkitbenchmarker.providers.rackspace import rackspace_virtual_machine  # NOQA
+import_util.LoadModulesWithName(__path__, __name__, 'flags')
```

Comment:
Could you document:
* The purpose of this.
* The expectations of provider subpackages?